docs/configure_open_edx.md

```diff
@@ -0,0 +1,57 @@
+Configure Open edX
+---
+
+In order to create user accounts in Open edX and permit authentication from xPro to Open edX, you need to configure xPro as an OAuth2 provider for Open edX.
+
+
+#### Setup Open edX Devstack
+
+Follow devstack's [README instructions](https://github.com/edx/devstack#getting-started) to get a functional devstack instance running.
+
+#### Install `social-auth-mitxpro` in LMS
+
+There are two options for this:
+
+##### Install via pip
+
+- `pip install social-auth-mitxpro`
+
+##### Install from local Build
+
+- Checkout the [social-auth-mitxpro](https://github.com/mitodl/social-auth-mitxpro) project and build the package per the project instructions
+- Copy the `social-auth-mitxpro-$VERSION.tar.gz` file into devstack's `edx-platform` directory
+- In devstack, run `make lms-shell` and within that shell `pip install social-auth-mitxpro-$VERSION.tar.gz`
+  - To update to a new development version without having to actually bump the package version, simply `pip uninstall social-auth-mitxpro`, then install again
+
+
+#### Configure xPro as a OAuth provider for Open edX
+
+In xPro:
+
+- go to `/admin/oauth2_provider/application/` and create a new application with these settings selected:
+  - `Redirect uris`: `http://$EDX_HOSTNAME:18000/auth/complete/mitxpro-oauth2/`
+  - `Client type`: "Confidental"
+  - `Authorization grant type`: "Authorization code"
+  - `Skip authorization`: checked
+  - Other values are arbitrary but be sure to fill them all out. Save the client id and secret for later
+
+In Open edX:
+- Follow instructions [here](https://edx.readthedocs.io/projects/edx-installing-configuring-and-running/en/latest/configuration/tpa/tpa_integrate_open/tpa_oauth.html#additional-oauth2-providers-advanced) on setting up the provider config with a few corrections:
```

Comment:
Yeah, I think the amount of instructions and the total difference is enough to justify putting our own instructions here instead of linking. For convenience here's my list of json file changes needed in devstack

```
# lms.env.json
FEATURES.ENABLE_THIRD_PARTY_AUTH = true
THIRD_PARTY_AUTH_BACKENDS = ["social_auth_mitxpro.backends.MITxProOAuth2"]

# lms.auth.json
SOCIAL_AUTH_OAUTH_SECRETS = {"mitxpro-oauth2": "<xpro_application_client_secret>"}
```